### DISCUSSION TITLE
Web UI stopped working

### DISCUSSION THREAD

**Discussion by jdecker79** [Security-Onion-Solutions/securityonion#5502] [Unsupported Versions] [answered]
Yesterday I was not able to access the web UI as I normally have using the IP Address of my Security Onion install. 
I am trying to access from Chrome - 93.0.4577.82 & Firefox 92.0.
When viewing Dev Tools I noticed items trying to load failing with net::ERR_HTTP2_PROTOCOL_ERROR.  
After multiple refreshes the home page finally loaded after 16.5 minutes.
I am also seeing the same errors when trying to open Grafana after the home page comes up.
The only changes I have made to SO in the last couple of weeks was to add syslogs from a switch, and two WAP.
The home page shows Version 2.3.20 in the bottom left corner so appears to be up to date. **(sorry just noticed manager is not updated, running update now and will advise if this corrects the issue.)**

**Updating to latest version did not solve the Web UI issue.**

Any suggestions on what could be causing this issue.

**Comment by dougburks** [comment, marked as answer]
Please include more information as described in https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

- Install source. Did you install from our Security Onion ISO image or did you perform a network installation?
- If network install, did you install from CentOS 7 or Ubuntu 18.04?
- Install type. ex. eval, standalone, forward node, search node, etc
- Does so-status show all services running?
- Do you get any failures when you run salt-call state.highstate from the node?
- Does the grid status show any failures?

What are the specs of the box?

What is the load average on the box?  

What is the volume of logs from the switch and WAPs?

**Reply by jdecker79** [reply]
Install Source was Security Onion ISO image.
Install type is Stand Alone
so-status shows all services running. Have rebooted multiple times.
When I ran salt-call state.highstate on the manager box I got the following result:
Suceeded: 645 (changed 25)
Failed: 0
Total State Runs: 645
Total Runtime: 107.467 s

When I connect to the Master box using putty on my laptop it took forever for the salt-call state.highstate to finish.  But the results were the same and said Total Runtime was 99.489 s.  
 
Box is a Dell Optiplex 9020 Quad Core i7 4770 with 32 GB Ram, 1 TB SSD

Not sure how to find load average.

Not sure on the volume of logs from switch and WAPS (Novice Beginner) but I did disable them two days ago, because that is the only change I had made to the system before this issue started.

**Reply by dougburks** [reply]
> Not sure how to find load average.

https://www.howtogeek.com/194642/understanding-the-load-average-on-linux-and-other-unix-like-systems/

**Reply by jdecker79** [reply]
Uptime = 13:21:12 up 23:59, 2 users, load average: 1.03, 0.99, 1.05
I am beginning to think that there is something within my network that is hampering the communication between my workstations and the Security Onion web server, because this is the only site I have issues accessing.  Normal internet access seems unaffected.

**Reply by dougburks** [reply]
Your load average doesn't seem to be high enough to cause the problem you're describing.

It sounds like you're on the right track looking at something on your network.

**Reply by jdecker79** [reply]
Just wanted to post an update.  For whatever reason the SOC is now working again.  Don't have the slightest idea what changed to make it start working again.
